src/main/resources/default.properties

```diff
@@ -6,3 +6,6 @@ JDBC_URL=127.0.0.1
 DATABASE_URL="test_url"
 DATABASE_PASSWORD="test_password"
 DATABASE_USERNAME=test_username
+BACKUP_FREQUENCY="daily"
+BACKUP_TIME="23:00"
```

Comment:
BACKUP_TIME is not sufficient for:
- weekly
  - needs day of the week
- monthly
  - needs day in the month
  - rounded down, so 31 becomes 28,29,30 or 31 depending on the month (& year)
  - UP FOR DEBATE:
    - first [specified_day] of the month
- yearly
  - needs dd-mm date

BACKUP_TIME is still needed for all these options of course